src/index.html

```diff
@@ -1,13 +1,64 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+  </head>
+
+  <body>
+    <div class="product-card">
+      <img
+        src="styles/images/imac.jpeg"
+        alt="imac"
+        class="product-card__picture-imac"
+      >
+      <p class="product-card__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
+      <p class="product-card__code">
+        Product code: 195434
+      </p>
+      <div class="stars" data-qa="card">
+        <div class="stars__content">
+          <div class="stars__stars-4">
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-active"></div>
+            <div class="stars__star-inactive"></div>
+          </div>
+          <div class="stars__text">
+            <p class="stars__text-reviews">Reviews: 5</p>
+          </div>
+        </div>
+      </div>
```

Comment:
1) `stars_content` is redundant here
2) do not mix up blocks, elements and modifiers names, also – they cannot be combined into one classname, all should be individual:
  	- block name – `block`
  	- element name – `block__element`
  	- modifier name – `block__element--modifier`
 3) no need to add `-active` and `-inactive` suffixes, just `stars_star`, you'll handle their image using styles
4) semantic tags is a must, you can use them
```
      <section class="product-card__review">
        <div class="stars stars--4">
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
          <div class="stars__star"></div>
        </div>
        <h3 class="product-card__feedbacks">
          Reviews: 5
        </h3>
      </section>
```